index.bs

```diff
@@ -5417,15 +5417,15 @@ same [=[RP]=] and passing through the same attacker.
 
 Such an attack would potentially be detectable; since the [=[RP]=] has registered the attacker's [=credential public key=] rather
 than the user's, the attacker must tamper with all subsequent [=authentication ceremonies=] with that [=[RP]=]: unscathed
-ceremonies will fail.
+ceremonies will fail, risking to reveal the attack.
 
 [=Attestation types=] other than [=Self Attestation=] and [=None=] can increase the difficulty of such attacks, since [=[RPS]=]
 can possibly display [=authenticator=] information, e.g., model designation, to the user. An attacker might therefore need to use
 a genuine [=authenticator=] of the same model as the user's [=authenticator=], or the user might notice that the [=[RP]=] reports
 a different [=authenticator=] model than the user expects.
 
 Note: All variants of [=man-in-the-middle attacks=] described above are more difficult for an attacker to mount
-than a [=man-in-the-middle attack=] against a [=[RP]=] employing conventional password authentication.
+than a [=man-in-the-middle attack=] against a [=[RP]=] employing conventional password authentication only.
```

Comment:
I dunno if I'd say "only" because RPs that adopt webauthn/fido are, it seems, likely to support legacy uname/password along with webauthn/fido for at least a while....

Reply:
Hm, yes, that's what I was thinking too... Do you read it as "more difficult only if the RP uses password auth"?

Maybe the "only" should be between "employing" and "conventional" instead:

>more difficult for an attacker to mount than a [=man-in-the-middle attack=] against a [=[RP]=] employing only conventional password authentication.

Reply:
I think we've identified the same ambiguity but are approaching it from opposite directions. :) So maybe a better solution is something like this:

>Note: All variants of [=man-in-the-middle attacks=] described above are more difficult for an attacker to mount than a [=man-in-the-middle attack=] against conventional password authentication.

Reply:
LGTM, thx!